frontend/src/ProjectorLoader.js

```diff
@@ -3,17 +3,32 @@ import React from "react"
 import styled from "styled-components";
 import loading from "./assets/loading.gif";
 
+
+const ratioHeight = 96/150;
+const ratioWidth = 150/96;
+
 const LoadingImage = styled.div`
     background-image: url(${props => props.src});
     background-size: cover;
-    height: 96px;
-    width: 150px;    
+    height: ${props => props.height};
+    width: ${props => props.width};    
     margin: 0 auto;
 `;
 
-const ProjectorLoader = () => (<div >
-  <LoadingImage src={loading} />
-</div>)
+
+const ProjectorLoader = ({height = '96px', width = '150px'}) => {
+
+  if(height !== '96px') {
+    width = (parseInt(height.replace('px', '')) * ratioWidth) + 'px';
+  } else if (width !== '96px') {
+    height = (parseInt(width.replace('px', '')) * ratioHeight) + 'px';
```

Comment:
I would move the formatting to CSS instead :)

Just make `width` and `height` numbers and format the CSS string like so:

```css
height: ${props => props.height}px;
```